leasing/serializers/land_area.py

```diff
@@ -386,6 +386,13 @@ class CustomDetailedPlanSerializer(
         allow_null=True,
         required=False,
     )
+    type = InstanceDictPrimaryKeyRelatedField(
+        instance_class=PlanUnitType,
+        queryset=PlanUnitType.objects.filter(),
```

Comment:
Just out of curiosity, does this filter() -function work without any parameters? Isn't objects.all() basically the same thing in this case?